vnext/Desktop.UnitTests/UtilsTest.cpp

```diff
@@ -25,7 +24,7 @@ TEST_CLASS(UtilsTest) {
       string path = "/",
       string query = "") {
 
-    Url url(urlString);
+    Url url(std::move(urlString));
```

Comment:
>std::move(urlString) [](start = 12, length = 20)

Can we take this one step further and declare the 'urlString' parameter as an r-value reference

Reply:
Sounds good, but being a backport, I prefer to change this in a subsequent, master-targeting PR.